packages/react/src/lib/accessibility/Behaviors/Alert/alertWarningBehavior.ts

```diff
@@ -1,17 +1,22 @@
 import { Accessibility } from '../../types'
+import AlertProps from './alertProps'
 
 /**
  * @specification
- * Adds role 'alert' to 'content' slot.
- * Adds attribute 'aria-live=polite' to 'content' slot.
+ * Adds role 'alert' to 'body' slot.
+ * Adds attribute 'aria-live=polite' to 'body' slot.
  */
 
-const alertWarningBehavior: Accessibility = () => ({
+const alertWarningBehavior: Accessibility<AlertProps> = props => ({
   attributes: {
-    content: {
+    body: {
       role: 'alert',
       'aria-live': 'polite',
     },
+    dismissAction: {
+      'aria-describedby': props.bodyId,
+      'aria-label': 'dismiss',
```

Comment:
Is it valid to have unlocalized defaults? @jurokapsiar

Reply:
no, please remove. User will need to provide this